x-pack/plugins/osquery/public/shared_components/osquery_action/index.tsx

```diff
@@ -20,36 +26,19 @@ import { useIsOsqueryAvailable } from './use_is_osquery_available';
 interface OsqueryActionProps {
   agentId?: string;
   formType: 'steps' | 'simple';
-  hideFullscreen?: true;
+  isExternal?: true;
```

Comment:
Is this type annotation correct? `isExternal?: true`. So it's either undefined or true?

Reply:
Thanks for pointing this out, I just had another discussion about it with @patrykkopycinski ;p 

in essence: I am going to get rid of this approach anyway in the next PR, and decided for the future to go with `x? : boolean` not to confuse anybody anymore :P 

However, my point here was to specify that we use this flag only as 'true' so we do not really have to pass default = false, nor x={false} everywhere. It's a specific type that says, that we either ignore this prop or set it to true in our edge cases. But I understand it may bring more confusion given the name is `is...` which may suggest a boolean :) 

@janmonschke however I would love to hear your opinion about this :) Thanks!

Reply:
Ah yes, I understand it now. It does feel rather confusing, given it starts with `is` 😅 
Let's go with booleans and defaults in the future which might be slightly less elegant but less confusing.